lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -13,6 +13,8 @@ ShopifyApp.configure do |config|
   config.api_key = ENV.fetch('SHOPIFY_API_KEY', '').presence
   config.secret = ENV.fetch('SHOPIFY_API_SECRET', '').presence
 
+  config.log_level = ENV["SHOPIFY_LOG_LEVEL"]&.to_sym || :info
```

Comment:
Should we just do something like
`config.log_level = :info` 
and let them edit the `shopify_app.rb` file directly?

Reply:
Yeah, that seems better, maybe we don't need the `SHOPIFY_LOG_LEVEL` setting at all.